src/edu/csus/ecs/pc2/clics/API202003/JSONTool.java

```diff
@@ -85,17 +84,15 @@ public ObjectNode convertToJSON(Run submission, HttpServletRequest servletReques
 
         // FIXME we need separate event feeds for public and admin/analyst
         // FIXME perhaps change sc to a boolean for public or not?
+        // JB: 12/24: Not sure why we need different ones for a submission.
         // if (servletRequest != null && (sc != null && sc.isUserInRole("admin") || sc.isUserInRole("analyst"))) {
 
-
-        // TODO shadow add time and mime elements to submission
```

Comment:
This deleted TODO refers to two things:  "add time" and "add mime".  I see that the latter was done; does the former still need to be retained as a "TODO"?